# Welcome

We develop novel tools and workflows for handling large amounts of sequence data. We apply computational methods to automate de novo (meta-)genome and transcriptome assembly, analysis and annotation of RNA-Seq data, identification and characterization of pathogens such as viruses and bacteria from environmental samples and (more recently) molecular monitoring of infectious diseases.

We strive to combine well-established and novel computational approaches into reproducible workflows to analyze and visualize large amounts of short- and long-read sequencing data.

## Research interests

* Microbial bioinformatics
* de novo assembly & annotation
* Transcriptomics & comparative genomics
* Metagenomics & -transcriptomics
* Real-time nanopore sequencing
* Workflow management & reproducibility
* Automation & visualization