# Publications
<p>This list is also available on <a href="https://scholar.google.de/citations?user=DMZ7Hc8AAAAJ">Google Scholar</a> and under <a href="https://orcid.org/0000-0001-7090-8717">ORCID</a>.</p>

<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.0 Transitional//EN" "http://www.w3.org/TR/REC-html40/loose.dtd">
<div class="publication"><ol>
<li value="49">
<b>DarkQ: Continuous genomic monitoring using message queues</b> (2021) A Viehweger, C Brandt, <span class="label">M H&ouml;lzer</span>. <em>F </em>1000Research 10 (998), 998</li>
<li value="48">
<b>Rise and Fall of SARS-CoV-2 Lineage A. 27 in Germany</b> (2021) S Calvignac-Spencer, M Budt, M Huska, H Richard, L Leipold, et al. <em>Viruses </em>13 (8), 1491</li>
<li value="47">
<b>Evidence for the existence of a new genus Chlamydiifrater gen. nov. inside the family Chlamydiaceae with two new species isolated from flamingo (<em>Phoenicopterus roseus</em>&nbsp;&hellip;</b> (2021) F Vorimore, <span class="label">M H&ouml;lzer</span>, EM Liebler-Tenorio, LM Barf, S Delannoy, et al. <em>Systematic and Applied Microbiology </em>44 (4), 126200</li>
<li value="46">
<b>Emergence of SARS-CoV-2 Lineage A. 27 in Germany, Expressing Viral Spike Proteins with Several Amino Acid Replacements of Interest, Including L18F, L452R, and N501Y in the&nbsp;&hellip;</b> (2021) S Calvignac-Spencer, M Budt, M Huska, H Richard, M von Kleist, et al. <em>Virological. org </em>15</li>
<li value="45">
<b>PoSeiDon: a Nextflow pipeline for the detection of evolutionary recombination events and positive selection</b> (2021) <span class="label">M H&ouml;lzer</span>, M Marz. <em>Bioinformatics </em>37 (7), 1018-1020</li>
<li value="44">
<b>SARS-CoV-2-Varianten: Evolution im Zeitraffer</b> () DY Oh, S Kr&ouml;ger, M Wedde, F Hartkopf, M Budt, J Seifried, A Radonic, et al. <em>Robert Koch-Institut </em>2021 <a href="" title="Citations"><span class="badge badge-inverse"><test>6x</test></span></a>
</li>
<li value="43">
<b>Computational strategies to combat COVID-19: useful tools to accelerate SARS-CoV-2 and coronavirus research</b> (2021) F Hufsky, K Lamkiewicz, A Almeida, A Aouacheria, C Arighi, A Bateman, et al. <em>Briefings in bioinformatics </em>22 (2), 642-663 <a href="" title="Citations"><span class="badge badge-inverse"><test>40x</test></span></a>
</li>
<li value="42">
<b>EpiDope: A Deep neural network for linear B-cell epitope prediction</b> (2021) M Collatz, F Mock, E Barth, <span class="label">M H&ouml;lzer</span>, K Sachse, M Marz. <em>Bioinformatics </em>37 (4), 448-455 <a href="" title="Citations"><span class="badge badge-inverse"><test>8x</test></span></a>
</li>
<li value="41">
<b>Metagenomics workflow for hybrid assembly, differential coverage binning, metatranscriptomics and pathway analysis (MUFFIN)</b> (2021) R Van Damme, <span class="label">M H&ouml;lzer</span>, A Viehweger, B M&uuml;ller, E Bongcam-Rudloff, et al. <em>PLOS Computational Biology </em>17 (2), e1008716 <a href="" title="Citations"><span class="badge badge-inverse"><test>8x</test></span></a>
</li>
<li value="40">
<b>A marine Chlamydomonas sp. emerging as an algal model</b> (2021) D Carrasco Flores, M Fricke, V Wesp, D Desir&ograve;, A Kniewasser, <span class="label">M H&ouml;lzer</span>, et al. <em>Journal of Phycology </em>57 (1), 54-69</li>
<li value="39">
<b>Rates of primary production in groundwater rival those in oligotrophic marine systems</b> () WA Overholt, S Trumbore, X Xu, TLV Bornemann, AJ Probst, M Kr&uuml;ger, et al. <em>bioRxiv </em>2021</li>
<li value="38">
<b>Adaptive nanopore sequencing on miniature flow cell detects extensive antimicrobial resistence</b> () A Viehweger, M Marquet, <span class="label">M H&ouml;lzer</span>, N Dietze, M Pletz, C Brandt. <em>bioRxiv </em>2021</li>
<li value="37">
<b>The economical lifestyle of CPR bacteria in groundwater allows little preference for environmental drivers</b> () NM Chaudhari, WA Overholt, PA Figueroa-Gonzalez, M Taubert, et al. <em>bioRxiv </em>2021</li>
<li value="36">
<b>Context-aware genomic surveillance reveals hidden transmission of a carbapenemase-producing Klebsiella pneumoniae</b> () A Viehweger, C Blumenscheit, N Lippmann, KL Wyres, C Brandt, JB Hans, et al. <em>bioRxiv </em>2021</li>
<li value="35">
<b>poreCov-an easy to use, fast, and robust workflow for SARS-CoV-2 genome reconstruction via nanopore sequencing</b> () C Brandt, S Krautwurst, R Spott, M Lohde, M Jundzill, M Marquet, et al. <em>bioRxiv </em>2021</li>
<li value="34">
<b>Chromosome-level genome assembly and transcriptome-based annotation of the oleaginous yeast Rhodotorula toruloides CBS 14</b> () GC Mart&iacute;n-Hern&aacute;ndez, B M&uuml;ller, M Chmielarz, C Brandt, <span class="label">M H&ouml;lzer</span>, et al. <em>bioRxiv </em>2021</li>
<li value="33">
<b>CovRadar: Continuously tracking and filtering SARS-CoV-2 mutations for molecular surveillance</b> () A Wittig, F Miranda, <span class="label">M H&ouml;lzer</span>, T Altenburg, JM Bartoszewicz, et al. <em>bioRxiv </em>2021</li>
<li value="32">
<b>RUNX3 Transcript Variants Have Distinct Roles in Ovarian Carcinoma and Differently Influence Platinum Sensitivity and Angiogenesis</b> (2021) K Heinze, <span class="label">M H&ouml;lzer</span>, M Ungelenk, M Gerth, J Thomale, R Heller, et al. <em>Cancers </em>13 (3), 476</li>
<li value="31">
<b>Molecular epidemiology of SARS-CoV-2-a regional to global perspective</b> () C Brandt, R Spott, <span class="label">M H&ouml;lzer</span>, D Kuehnert, S Fuchs, M Lohde, M Marquet, et al. <em>medRxiv </em>2021</li>
<li value="30">
<b>SLC35F2, a Transporter Sporadically Mutated in the Untranslated Region, Promotes Growth, Migration, and Invasion of Bladder Cancer Cells</b> (2021) R Kotolloshi, <span class="label">M H&ouml;lzer</span>, M Gajda, MO Grimm, D Steinbach. <em>Cells </em>10 (1), 80</li>
<li value="29">
<b>Virus Bioinformatics</b> (2021) N Pappas, S Roux, <span class="label">M H&ouml;lzer</span>, K Lamkiewicz, F Mock, M Marz, BE Dutilh. <em>Encyclopedia of Virology </em>124</li>
<li value="28">
<b>A decade of <em>de novo</em> transcriptome assembly: Are we there yet?</b> (2021) <span class="label">M H&ouml;lzer</span>. <em>Molecular Ecology Resources </em>21 (1), 11-13</li>
<li value="27">
<b>RNAflow: An Effective and Simple RNA-Seq Differential Gene Expression Pipeline Using Nextflow</b> (2020) M Lataretu, <span class="label">M H&ouml;lzer</span>. <em>Genes </em>11 (12), 1487</li>
<li value="26">
<b>Comparative Genome Analysis of 33 Chlamydia Strains Reveals Characteristic Features of Chlamydia Psittaci and Closely Related Species</b> (2020) <span class="label">M H&ouml;lzer</span>, LM Barf, K Lamkiewicz, F Vorimore, M Lataretu, A Favaroni, et al. <em>Pathogens </em>9 (11), 899 <a href="" title="Citations"><span class="badge badge-inverse"><test>6x</test></span></a>
</li>
<li value="25">
<b>Inclusion of Oxford Nanopore long reads improves all microbial and viral metagenome&#8208;assembled genomes from a complex aquifer system</b> (2020) WA Overholt, <span class="label">M H&ouml;lzer</span>, P Geesink, C Diezel, M Marz, K K&uuml;sel. <em>Environmental Microbiology </em>22 (9), 4000-4013 <a href="" title="Citations"><span class="badge badge-inverse"><test>17x</test></span></a>
</li>
<li value="24">
<b>A comprehensive annotation and differential expression analysis of short and long non-coding RNAs in 16 bat genomes</b> (2020) NF Mostajo, M Lataretu, S Krautwurst, F Mock, D Desir&ograve;, K Lamkiewicz, et al. <em>NAR Genomics and Bioinformatics </em>2 (1), lqz006</li>
<li value="23">
<b>What the Phage: A scalable workflow for the identification and analysis of phage sequences</b> () M Marquet, <span class="label">M H&ouml;lzer</span>, MW Pletz, A Viehweger, O Makarewicz, R Ehricht, et al. <em>bioRxiv </em>2020</li>
<li value="22">
<b>Virus-and Interferon Alpha-Induced Transcriptomes of Cells from the Microbat <em>Myotis daubentonii</em></b> (2019) <span class="label">M H&ouml;lzer</span>, A Schoen, J Wulle, MA M&uuml;ller, C Drosten, M Marz, F Weber. <em>iScience </em>19, 647-661 <a href="" title="Citations"><span class="badge badge-inverse"><test>12x</test></span></a>
</li>
<li value="21">
<b><em>Chlamydia buteonis</em>, a new Chlamydia species isolated from a red-shouldered hawk</b> (2019) K Laroucau, F Vorimore, R Aaziz, L Solmonson, RC Hsia, PM Bavoil, et al. <em>Systematic and applied microbiology </em>42 (5), 125997 <a href="" title="Citations"><span class="badge badge-inverse"><test>35x</test></span></a>
</li>
<li value="20">
<b>Direct RNA nanopore sequencing of full-length coronavirus genomes provides novel insights into structural variants and enables modification analysis</b> (2019) A Viehweger, S Krautwurst, K Lamkiewicz, R Madhugiri, J Ziebuhr, et al. <em>Genome research </em>29 (9), 1545-1554 <a href="" title="Citations"><span class="badge badge-inverse"><test>127x</test></span></a>
</li>
<li value="19">
<b>Evaluation of Sequencing Library Preparation Protocols for Viral Metagenomic Analysis from Pristine Aquifer Groundwaters</b> (2019) R Kallies, <span class="label">M H&ouml;lzer</span>, R Brizola Toscan, U Nunes da Rocha, J Anders, et al. <em>Viruses </em>11 (6), 484 <a href="" title="Citations"><span class="badge badge-inverse"><test>17x</test></span></a>
</li>
<li value="18">
<b><em>De novo</em> transcriptome assembly: A comprehensive cross-species comparison of short-read RNA-Seq assemblers</b> (2019) <span class="label">M H&ouml;lzer</span>, M Marz. <em>GigaScience </em>8 (5), giz039 <a href="" title="Citations"><span class="badge badge-inverse"><test>78x</test></span></a>
</li>
<li value="17">
<b>Structure and hierarchy of influenza virus models revealed by reaction network analysis</b> (2019) S Peter, <span class="label">M H&ouml;lzer</span>, K Lamkiewicz, PS Di Fenizio, H Al Hwaeer, M Marz, et al. <em>Viruses </em>11 (5), 449 <a href="" title="Citations"><span class="badge badge-inverse"><test>10x</test></span></a>
</li>
<li value="16">
<b>SilentMutations (SIM): A tool for analyzing long-range RNA&ndash;RNA interactions in viral genomes and structured RNAs</b> (2019) D Desir&ograve;, <span class="label">M H&ouml;lzer</span>, B Ibrahim, M Marz. <em>Virus research </em>260, 135-141</li>
<li value="15">
<b>Cohesin-mediated NF-&kappa;B signaling limits hematopoietic stem cell self-renewal in aging and inflammation</b> (2019) Z Chen, EM Amro, F Becker, <span class="label">M H&ouml;lzer</span>, SMM Rasa, SN Njeru, B Han, et al. <em>Journal of Experimental Medicine </em>216 (1), 152-175 <a href="" title="Citations"><span class="badge badge-inverse"><test>43x</test></span></a>
</li>
<li value="14">
<b>Addressing dereplication crisis: Taxonomy-free reduction of massive genome collections using embeddings of protein content</b> (2019) A Viehweger, M Hoelzer, C Brandt. <em>bioRxiv </em>855262</li>
<li value="13">
<b>PCAGO: An interactive tool to analyze RNA-Seq data with principal component analysis</b> (2019) R Gerst, <span class="label">M H&ouml;lzer</span>. <em>bioRxiv </em>433078 <a href="" title="Citations"><span class="badge badge-inverse"><test>8x</test></span></a>
</li>
<li value="12">
<b>Analysis of molecular mechanism of progression of non-muscle-invasive bladder cancer (NMIBC) by genome-wide exome and UTR mutation analysis</b> (2018) D Steinbach, <span class="label">M H&ouml;lzer</span>, M Marz, M Gajda, FC Von Rundstedt, MO Grimm. <em>European Urology Supplements </em>17 (2), e1523</li>
<li value="11">
<b>Complete Genome Sequence of JII-1961, a Bovine Mycobacterium avium subsp. paratuberculosis Field Isolate from Germany</b> (2017) P M&ouml;bius, G Nordsiek, <span class="label">M H&ouml;lzer</span>, M Jarek, M Marz, H K&ouml;hler. <em>Genome announcements </em>5 (34), e00870-17 <a href="" title="Citations"><span class="badge badge-inverse"><test>7x</test></span></a>
</li>
<li value="10">
<b>Evaluation of associations between genotypes of Mycobacterium avium subsp. paratuberculsis and presence of intestinal lesions characteristic of paratuberculosis</b> (2017) P M&ouml;bius, E Liebler-Tenorio, <span class="label">M H&ouml;lzer</span>, H K&ouml;hler. <em>Veterinary Microbiology </em>201, 188-194</li>
<li value="9">
<b>Massive effect on lncRNAs in human monocytes during fungal and bacterial infections and in response to vitamins A and D</b> (2017) K Riege, <span class="label">M H&ouml;lzer</span>, TE Klassert, E Barth, J Br&auml;uer, M Collatz, F Hufsky, et al. <em>Scientific reports </em>7, 40598 <a href="" title="Citations"><span class="badge badge-inverse"><test>22x</test></span></a>
</li>
<li value="8">
<b>Differential effects of vitamins A and D on the transcriptional landscape of human monocytes during infection</b> (2017) TE Klassert, J Br&auml;uer, <span class="label">M H&ouml;lzer</span>, M Stock, K Riege, C Zubir&iacute;a-Barrera, et al. <em>Scientific reports </em>7 (1), 1-15 <a href="" title="Citations"><span class="badge badge-inverse"><test>15x</test></span></a>
</li>
<li value="7">
<b>The Dark Art of Next-generation Sequencing: Fundamental Approaches for Genomics, Transcriptomics, and Differential Gene Expression</b> () <span class="label">M H&ouml;lzer</span>. <em>Friedrich-Schiller-Universit&auml;t Jena </em>2017</li>
<li value="6">
<b>Software Dedicated to Virus Sequence Analysis "Bioinformatics Goes Viral"</b> (2017) <span class="label">M H&ouml;lzer</span>, M Marz. <em>Advances in Virus Research </em>99, 233-257 <a href="" title="Citations"><span class="badge badge-inverse"><test>18x</test></span></a>
</li>
<li value="5">
<b>Evolution and antiviral specificity of interferon-induced Mx proteins of bats against Ebola-, Influenza-, and other RNA viruses</b> () J Fuchs, <span class="label">M H&ouml;lzer</span>, A Schilling, Mirjam: Patzina, Corinna: Schoen, et al. <em>Journal of Virology </em>2017 <a href="" title="Citations"><span class="badge badge-inverse"><test>41x</test></span></a>
</li>
<li value="4">
<b>Differential transcriptional responses to Ebola and Marburg virus infection in bat and human cells</b> (2016) <span class="label">M H&ouml;lzer</span>, V Kr&auml;hling, F Amman, E Barth, SH Bernhart, VAO Carmelo, et al. <em>Scientific reports </em>6 (1), 1-17 <a href="" title="Citations"><span class="badge badge-inverse"><test>42x</test></span></a>
</li>
<li value="3">
<b>Whole-Genome Sequence of Chlamydia gallinacea Type Strain 08-1274/3</b> (2016) <span class="label">M H&ouml;lzer</span>, K Laroucau, HH Creasy, S Ott, F Vorimore, PM Bavoil, M Marz, et al. <em>Genome announcements </em>4 (4), e00708-16 <a href="" title="Citations"><span class="badge badge-inverse"><test>8x</test></span></a>
</li>
<li value="2">
<b>Comprehensive insights in the Mycobacterium avium subsp. paratuberculosis genome using new WGS data of sheep strain JIII-386 from Germany</b> (2015) P M&ouml;bius, <span class="label">M H&ouml;lzer</span>, M Felder, G Nordsiek, M Groth, H K&ouml;hler, et al. <em>Genome biology and evolution </em>7 (9), 2585-2601 <a href="" title="Citations"><span class="badge badge-inverse"><test>18x</test></span></a>
</li>
<li value="1">
<b>Towards a comprehensive picture of alloacceptor tRNA remolding in metazoan mitochondrial genomes</b> (2015) AH Sahyoun, <span class="label">M H&ouml;lzer</span>, F J&uuml;hling, C H&ouml;ner zu Siederdissen, M Al-Arab, et al. <em>Nucleic acids research </em>43 (16), 8044-8056 <a href="" title="Citations"><span class="badge badge-inverse"><test>17x</test></span></a>
</li>
</ol></div>
